### ISSUE TITLE
Sandstorm: One of my spreadsheets can no longer scroll

### ISSUE THREAD

**Issue by ocdtrekkie** [audreyt/ethercalc#453]
So, really weird behavior on this one. I have a spreadsheet that won't scroll vertically anymore, and like... the Sandstorm frame has an extra set of scrollbars, and an additional weird scrollbar type thing if you scroll down below the EtherCalc content. It's really odd, and I can confirm it doesn't appear on a new grain. The grain that has the issue has some heavily expanded columns and rows, size-wise, so I think something with that is what's messing with the rendering.

It's a private-ish grain, but I can share it with someone if they want to investigate it.

**Comment by eddyparkinson** [comment]
I tried adding wide cols http://sheet.cellmaster.com.au/o8o4zuf58lpz 
but all was fine. 

Maybe use the clipboard tab to copy the content to a new sheet.

If you still get the error, try to copy just 50% of the clipboard contents.

**Comment by ocdtrekkie** [comment]
There's a good chance it either: A. Only occurs in MS Edge, as I haven't yet tested other browsers. Or B. Only occurs in Sandstorm's frame, as that does add more complexity. I'll have to do a bit more testing to narrow it down.

**Comment by ocdtrekkie** [comment]
I am getting this behavior in both Edge and Firefox, but I do suspect it is specific to the Sandstorm frame.

**Comment by ocdtrekkie** [comment]
Tagging @paulproteus from the Sandstorm team

**Comment by lvign** [comment]
I have the same problem with a spreadsheet. I can copy and paste in the "non accessible/hidden portion", but I can not "navigate" in this "non accessible/hidden portion" of my spreadsheet. I am using Firefox for Ubuntu (Firefox version: 50.1.0). 
I managed to export it and everything is exported correctly, but this spreadsheet is not more usable.

**Comment by eddyparkinson** [comment]
@lvign Have you an example of what you mean by "non accessible/hidden portion".
Do you mean your spreadsheet no longer scrolls?

**Comment by lvign** [comment]
Yes, exactly, it doesn't scroll anymore (it worked at the beginning)

**Comment by eddyparkinson** [comment]
Without an example, it is hard to see what has gone wrong. A link to an example, or a copy of the social calc clipboard data would be good. 

If two of you have had the problem, that suggests there are others.

**Comment by ocdtrekkie** [comment]
@eddyparkinson I am happy to share the grain with you off-GitHub so you can see the behavior I'm looking at. But it's a relatively private grain, so I don't want to post screenshots or a share link on GitHub, and I don't know how to reach you outside of GitHub.

**Comment by eddyparkinson** [comment]
**scroll lock - bug**
@Ivign - The cause is the scroll lock, bottom left of spreadsheet the "drag to lock ...", the blue bar.

Moving the blue bar allows you to scroll as normal again.  (worked with your sheet)

**regression**
I think I fixed this bug about 2 years ago, so not sure why you are still getting it as the code you have looks newer. If I remember, a hide or insert or something causes control.paneslider to be given an invalid value and moving it restores thing back to normal.

**Comment by lvign** [comment]
Many thanks ! 
Laurence

**Comment by eddyparkinson** [comment]
@ocdtrekkie is this the same bug,i.e if you locate the blue scroll lock bar, and move it. The bar was off the screen to the far right. Does it fix things? if not, please share with eddyparkinson after the at add gmail.com.

**Comment by ocdtrekkie** [comment]
Heh, I guess it was. The blue bar didn't appear until I scrolled right enough for the width of the rows to decrease. And I certainly didn't move the lock bar myself. Any idea how we can make this behavior less problematic in the future?

**Comment by ocdtrekkie** [comment]
FWIW, the EtherCalc version I am using appears to be dated 2016.06.12.0, so if it's better in the last few months, I wouldn't know.
